app/src/main/java/ru/vssemikoz/newsfeed/adapters/NewsFeedAdapter.java

```diff
@@ -71,9 +71,11 @@ public void onBindViewHolder(@NonNull NewsViewHolder holder, int position) {
         }
 
         if (newsItem.getImageUrl() != null) {
-            Picasso.with(context)
-                    .load(newsItem.getImageUrl())
-                    .into(holder.imageView);
+            if(!newsItem.getImageUrl().isEmpty()){
```

Comment:
Do we really need nested checks here? I'd like to make them one-liner. BTW, there's a TextUtils.isEmpty() that can replace those checks.